validator_client/src/notifier.rs

```diff
@@ -54,6 +54,23 @@ pub fn spawn_notifier<T: EthSpec>(client: &ProductionValidatorClient<T>) -> Resu
                         "epoch" => format!("{}", epoch),
                         "slot" => format!("{}", slot),
                     );
+
+                    if total_validators <= 5 {
+                        let mut validators = duties_service.attesters_for_epoch(epoch);
+                        validators.sort_by(|a, b| {
+                            let a = a.as_hex_string();
+                            let b = b.as_hex_string();
+                            a.cmp(&b)
+                        });
+                        for (i, validator) in validators.iter().enumerate() {
+                            info!(
+                                log_2,
+                                "Validators' public key";
+                                "public_key" => format!("{:?}", validator),
+                                "number" => i,
+                            );
+                        }
+                    }
```

Comment:
1. Not sure about the format. I wanted to add a "number" so that users wouldn't get lost, but since we iterate over a hashmap, we need to store the vector returned by attesters_for_epoch. Not too computationally heavy since it's at most 5 iterations but still... I'm not very satisfied with this code.
2. Is there a way to "skip" inside a for_each loop? I believe it would be better if we could only print the keys say 1/4th of the time.